docs/adr-0004-versioning.md

```diff
@@ -0,0 +1,75 @@
+# ADR 4: Versioning 5/1/2019
```

Comment:
nit: -> Release versioning? Just to clarify from gecko, git, or Fire OS versioning, or something.

I'd update the filename too.